Makefile.core.mk

```diff
@@ -97,14 +278,44 @@ prometheus/metrics.proto:
 prometheus/metrics.pb.go: prometheus/metrics.proto
 	@$(PROTOC) $(GOGOFASTER_PLUGIN):. prometheus/metrics.proto
 
-generate: $(googleapis_packages) $(census_packages) prometheus/metrics.pb.go
+k8sapi_prep:
+	@mkdir -p k8sapi/k8s.io/api
+	@cd k8sapi/k8s.io/api && mkdir -p $(k8sapi_packages)
+
+$(k8sapi_protos): %:
```

Comment:
ok learning time - whats `%:` do? :)

Reply:
So many questions...

This was cut & paste, I don't know for sure what the semantics are.